src/CurrencyChanger

```diff
@@ -0,0 +1,96 @@
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.text.DecimalFormat;
+import java.util.HashMap;
+import java.util.Map;
+
+public class CurrencyBalanceApp implements User {
+    private JFrame frame;
+    private JPanel mainPanel;
+    private JPanel balancePanel;
+    private JPanel currencyPanel;
+    private JPanel backPanel;
+    private JComboBox<String> currencyComboBox;
+    private JLabel balanceLabel;
+
+    private Map<String, Double> currencyRates;
+
+    private double currentBalance;
+
+    public CurrencyBalanceApp(monthlySavings){
```

Comment:
Parameters for need data types.  If you were to extend the user class, you would have access to the variable monthlysavings, but monthlySavings does not exist.  
But, see the comment above about extending user.